finance-tracker/app/controllers/user_stocks_controller.rb

```diff
@@ -1,2 +1,12 @@
 class UserStocksController < ApplicationController
+  def create
+    stock = Stock.find_by(ticker: (params[:ticker]))
+    if stock.blank?
+      stock = Stock.new_lookup(params[:ticker])
+      stock.save
+    end
+    stock.users << current_user
```

Comment:
`stock` will get error if line 6 returns false (fail to save) for whatever reason.